### DISCUSSION TITLE
[Android] Starting Service Remotely If Stopped

### DISCUSSION THREAD

**Discussion by stephaniehe8080** [rustdesk/rustdesk#8972] [General]
When I do adb shell am startservice -n com.carriez.flutter_hbb/.MainService , I receive Error: Requires permission not exported from uid 10674. 
Is there a way to start the Rustdesk service remotely as in unable to physically operate phone in case it stops for whatever reason?

**Comment by basilgello** [comment]
`adb shell am broadcast -a com.carriez.flutter_hbb.DEBUG_BOOT_COMPLETED com.carriez.flutter_hbb` if `Start on boot (beta)` is enabled.
